src/test/test-models/good/int_overloads.stan

```diff
@@ -0,0 +1,194 @@
+data { 
+  int d_int;
+  real d_real;
+}
+
+transformed data {
+  int transformed_data_int;
+  real transformed_data_real;
+
+  transformed_data_real = abs(d_int);
+  transformed_data_real = abs(d_real);
+  transformed_data_real = fabs(d_int);
+  transformed_data_real = fabs(d_real);
+
+  transformed_data_real = exp(d_int);
+  transformed_data_real = exp(d_real);
+  transformed_data_real = exp2(d_int);
+  transformed_data_real = exp2(d_real);
+  transformed_data_real = expm1(d_int);
+  transformed_data_real = expm1(d_real);
+  transformed_data_real = log(d_int);
+  transformed_data_real = log(d_real);
+  transformed_data_real = log1p(d_int);
+  transformed_data_real = log1p(d_real);
+  transformed_data_real = log2(d_int);
+  transformed_data_real = log2(d_real);
+  transformed_data_real = log10(d_int);
+  transformed_data_real = log10(d_real);
+
+  transformed_data_real = pow(d_int);
```

Comment:
How does this compile?  `pow` requires two arguments, so we need all permutations.

Reply:
Ah, I was too qucik there.

Reply:
Fixed.